templates/webapi/layouts/navbar.html.ep

```diff
@@ -7,11 +7,13 @@
      <div class="collapse navbar-collapse" id="navbarNav">
        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
          <li class='nav-item' id="all_tests">
-            %= link_to 'All Tests' => url_for('tests') => class => 'nav-link'
+            %= link_to 'All Tests' => url_for('tests') => class => 'nav-link', title => 'Lists all tests grouped by state'
          </li>
 
          <li class="nav-item dropdown" id="job_groups">
-            <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false" data-submenu>Job Groups</a>
+            <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button"
+               aria-haspopup="true" aria-expanded="false" data-submenu
+               title="Shows test results for a specific group (usually a specific distribution/product)">Job Groups</a>
```

Comment:
```suggestion
               title="Shows test results for a specific group (usually a specific product)">Job Groups</a>
```

I would avoid the word "distribution" as in the context of openQA it is mostly used to mean "test distribution", e.g. os-autoinst-distri-example, which does not usually correspond to job groups.